src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -38,15 +42,39 @@ def __init__(self):
     for extension in extensions_to_strip:
       app_name = utils.strip_from_right(app_name, extension)
 
-    self.trials = list(
+    db_trials = list(
         data_types.Trial.query(data_types.Trial.app_name == app_name))
 
+    self.trials = OrderedDict()
+    for trial in db_trials:
+      self.trials[trial.app_args] = trial.probability
+
+    trials_config_file = environment.get_value('SOURCE_SIDE_TRIALS')
```

Comment:
Would prefer we just enforce a condition with a hardcoded filename rather than introduce another variable. Would that work? 

E.g. something like `TRIALS_CONFIG_FILENAME = 'trials.json'`

Reply:
I have fixed the file name to `clusterfuzz_trials_config.json`